spec/acceptance/pip_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper_acceptance'
+
+describe 'python::pip defined resource' do
+  context 'install package with custom name' do
+    it 'works with no errors' do
+      pp = <<-PUPPET
+      class { 'python':
+        version => '3',
+        dev     => 'present',
+      }
+
+      python::pip { 'agent package':
+        pkgname => 'agent',
+        ensure  => '0.1.2',
+      }
+      PUPPET
+
+      apply_manifest(pp, catch_failures: true)
+      apply_manifest(pp, catch_changes: true)
+  end
```

Comment:
I think between line 19 and 20 is one `end` missing?

Reply:
Mostlikely, I'll check in the evening. Is there a way to run the acceptance tests locally?